apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-customization.tsx

```diff
@@ -144,6 +149,8 @@ export const SignInMethodCustomization: FunctionComponent<SignInMethodCustomizat
 
     const [ validationResult, setValidationResult ] =
         useState<ConnectionsJITUPConflictWithMFAReturnValue | undefined>(undefined);
+    const [ smsValidationResult, setSmsValidationResult ] =
+        useState<FederatedConflictWithSMSOTPReturnValue | undefined>(undefined);
```

Comment:
Shal we use `null` instead of undefined here and remove undefined from the type?

Reply:
Resolves via [43c7195](https://github.com/wso2/identity-apps/pull/3782/commits/43c71954ca90804c2445db3cf260b16ebcfa26c9)